### DISCUSSION TITLE
Welcome to blind_watermark Discussions!

### DISCUSSION THREAD

**Discussion by guofei9987** [guofei9987/blind_watermark#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lhwhit** [comment]
您好，我运行您的程序时出现了错误，刚接触Python，不知如何调试。错误信息如下：
(blind) os@os:~/blind_watermark/examples$ blind_watermark --embed -p 1x1 pic/ori_img.jpg pic/watermark.png output/embedded.png
{'work_mode': 'embed', 'password': '1x1', 'wm_shape': None}
['pic/ori_img.jpg', 'pic/watermark.png', 'output/embedded.png']
Traceback (most recent call last):
  File "/home/os/.local/bin/blind_watermark", line 8, in <module>
    sys.exit(main())
  File "/home/os/.local/lib/python3.8/site-packages/blind_watermark/cli_tools.py", line 32, in main
    bwm1.embed(args[2])
  File "/home/os/.local/lib/python3.8/site-packages/blind_watermark/blind_watermark.py", line 97, in embed
    self.idx_shuffle = np.random.RandomState(self.password_img) \
AttributeError: 'mtrand.RandomState' object has no attribute 'random'

**Reply by guofei9987** [reply]
numpy 版本过低，更新一下

**Comment by tdc6597135** [comment]
剪加水印的图片，剪裁后，不能提取出过水印的，是乱码，请教一下这个解决办法
